packages/application-shell/src/components/project-container/project-container.tsx

```diff
@@ -39,7 +39,7 @@ type TProjectContainerProps = {
   user: TFetchLoggedInUserQuery['user'];
   environment: TProviderProps<{ enableSignUp?: boolean }>['environment'];
   disableRoutePermissionCheck?: boolean;
-  render?: () => JSX.Element;
+  render?: () => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.